azext_edge/tests/edge/support/create_bundle_int/test_arccontainerstorage_int.py

```diff
@@ -0,0 +1,49 @@
+# coding=utf-8
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+
+from knack.log import get_logger
+from azext_edge.edge.common import OpsServiceType
+from azext_edge.edge.providers.edge_api import ARCCONTAINERSTORAGE_API_V1
+from .helpers import (
+    check_custom_resource_files,
+    check_workload_resource_files,
+    get_file_map,
+    run_bundle_command,
+)
+
+logger = get_logger(__name__)
+
+
+def test_create_bundle_arccontainerstorage(init_setup, tracked_files):
+    """Test for ensuring file names and content. ONLY CHECKS arccontainerstorage."""
+    # dir for unpacked files
+    ops_service = OpsServiceType.arccontainerstorage.value
+    command = f"az iot ops support create-bundle --ops-service {ops_service}"
+    walk_result, bundle_path = run_bundle_command(command=command, tracked_files=tracked_files)
+    file_map = get_file_map(walk_result, ops_service)["acs"]
+
+    check_custom_resource_files(file_objs=file_map, resource_api=ARCCONTAINERSTORAGE_API_V1)
+
+    expected_workload_types = ["daemonset", "deployment", "pod", "pvc", "replicaset", "service"]
+    expected_types = set(expected_workload_types).union(ARCCONTAINERSTORAGE_API_V1.kinds)
+    assert set(file_map.keys()).issubset(set(expected_types))
+    check_workload_resource_files(
+        file_objs=file_map,
+        expected_workload_types=expected_workload_types,
+        prefixes=[
+            "esa-otel-collector",
+            "csi-wyvern-controller",
+            "csi-wyvern-node",
+            "config-operator",
+            "edgevolume-mounthelper",
+            "w-adr-schema-registry",
+            "wyvern-operator",
+            "adr-schema-registry-cache-claimsrv",
+            "adr-schema-registry-cache-claim-user-pvc",
```

Comment:
Assuming these ADR elements are applicable only when schema registry is deployed?

Reply:
update to only check those prefixes when the schema registry pods under AIO are detected